### DISCUSSION TITLE
Is there any way to have a API for creating, deleting, updating sites ?

### DISCUSSION THREAD

**Discussion by Tdnshah** [vitodeploy/vito#294] [General]
Hi All,

Is there currently an REST API to manage billing related things with vito, if not is this on the plan or roadmap.
Happy to contribute as well.

Thank You 
Regards

Tejas D. Shah
Contact : +91-7738678091

**Comment by saeedvaziry** [comment]
VitoDeploy is a FREE software and doesn't have a billing system. and its not gonna have it.
